src/navs/index.js

```diff
@@ -3483,19 +3483,128 @@ export const docsMenu = {
                     icon: 'IconHome',
                     color: 'seagreen',
                 },
+                {
+                    name: 'Getting started',
+                },
+                {
+                    name: 'How it works',
+                    url: '/docs/error-tracking/how-it-works',
+                    icon: 'IconListCheck',
+                    featured: true,
+                    color: 'orange',
+                },
                 {
                     name: 'Installation',
                     url: '/docs/error-tracking/installation',
                     icon: 'IconBook',
                     color: 'blue',
                     featured: true,
+                    children: [
```

Comment:
One for another day I think but it'd be cool for the framework selection to be a global concept within our docs that you can toggle elsewhere and persists across all products / pages. This menu basically wouldn't need to exist in that case